packages/content/src/metadata.ts

```diff
@@ -0,0 +1,28 @@
+import { getItemMetadata } from "@esri/arcgis-rest-portal";
+import { IHubRequestOptions } from "@esri/hub-common";
+import { parse } from "fast-xml-parser";
```

Comment:
Is this what we use in the indexer @mikeringrose ?

Reply:
it is, though this PR is using v3.x, but indexer uses 2.x, and there are a few differences in the configuration settings and default behavior between the versions. I've made the equivalent configuration in this PR, but there are still slight differences in how the XML is parsed. For example, the [`peXml` node](https://github.com/Esri/hub.js/pull/375/files#diff-720828cca6aaaac8de88dda7278ca80fR35) is not parsed by v2 b/c it contains a nested XML document, but v3 parses that XML in place. See https://github.com/Esri/hub.js/pull/375/files#diff-720828cca6aaaac8de88dda7278ca80fR317-R324

Reply:
This is what we use. I wish we would have turned to it into nicer json, but it is what it is.